pkg/scripts/bigchaindb-monit-config

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+
+set -o nounset
+
+# Check if directory for monit logs exists
+if [ ! -d "$HOME/.bigchaindb-monit" ]; then
+  mkdir -p "$HOME/.bigchaindb-monit"
+fi
+
+monit_pid_path=${MONIT_PID_PATH:=$HOME/.bigchaindb-monit/monit_processes}
+monit_script_path=${MONIT_SCRIPT_PATH:=$HOME/.bigchaindb-monit/monit_script}
+monit_log_path=${MONIT_LOG_PATH:=$HOME/.bigchaindb-monit/logs}
+monit_exec_path=${MONIT_EXEC_PATH:=$HOME/.monitrc}
+
+function usage() {
+        cat <<EOM
+
+    Usage: ${0##*/} [-h]
+
+    Configure Monit for BigchainDB and Tendermint process management.
+
+    ENV[MONIT_PID_PATH] || --monit-pid-path PATH
+
+    Absolute path to directory where the the program's pid-file will reside. 
+    The pid-file contains the ID(s) of the process(es). (default: ${monit_pid_path})
+
+    ENV[MONIT_SCRIPT_PATH] || --monit-script-path PATH
+
+    Absolute path to the directory where the executable program or
+    script is present. (default: ${monit_script_path})
+
+    ENV[MONIT_LOG_PATH] || --monit-log-path PATH
+
+    Absolute path to the directory where all the logs for processes
+    monitored by Monit are stored. (default: ${monit_log_path})
+
+    ENV[MONIT_EXEC_PATH] || --monit-exec-path PATH
+
+    Absolute path to the directory to run the script form. (default: ${monit_exec_path})
```

Comment:
This information is outdated. It is now "the path to the Monit control file". I would also use a different variable name for it, like `monitrc_path`.

Reply:
done.